content/docs/how-we-work/task-estimation/estimations-and-story-points.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Estimations and Story Points
+weight: 3
+description: Estimates or estimations are commonly used terms during sprint planning or project planning exercises. They define the size of the work needed to achieve certain acceptance criteria or milestones in a project. It is essential to be explicit about the units of estimation being used—whether story points or hours—as this can cause confusion (e.g., 8 story points is not the same as 8 hours).
+---
+
+# What are Estimates?
+
+"Estimates" or estimations are commonly used terms during sprint planning or project planning exercises. They define the size of the work needed to achieve certain acceptance criteria or milestones in a project. It is essential to be explicit about the units of estimation being used—whether story points or hours—as this can cause confusion (e.g., 8 story points is not the same as 8 hours).
+
+## What are Story Points and How Are They Different from Hours?
+
+Story points are units of measure for expressing an estimate of the overall effort required to fully implement a product backlog item or any other piece of work. Teams assign story points relative to work complexity, the amount of work, and risk or uncertainty.
```

Comment:
@swarad07 I agree that SP is the recommended way however we are not using it as mentioned above in at least CDM and I believe it is same in Managed as well. 

Let us know your thoughts on this

Reply:
@Yemaneberhan-Lemma Same as above, lets not make that call of one over another, lets present what is recommended and mention that it alright to use hours.